app/models/config.rb

```diff
@@ -137,6 +142,10 @@ def self.shared_reset
     shared_reset_days.to_i
   end
 
+  def self.hide_budget_bar?
```

Comment:
If you wanted to add a test for that it would be nice, but not necessary - you can probably crib off the hide practical support test if it exists. should be in `tests/models/config_test.rb`.